### PR TITLE
Update runtime configuration

### PR DESCRIPTION
**Pull request by drgora**
This PR updates the runtime configuration to increase the maximum number of authorities and voters that elections can handle.

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -394,11 +395,11 @@ parameter_types! {
     pub HistoryDepth: u32 = 30; // Number of eras to keep in history. Older eras cannot be claimed.
 }
 
-pub const MAX_AUTHORITIES: u32 = 10;
-pub const MAX_VOTERS: u32 = 10;
+pub const MAX_AUTHORITIES: u32 = 32;
```
Comment (DanieleDiBenedetto) [inline runtime/src/lib.rs:398]: Duplication here with MaxActiveValidators ?

**Review by DanieleDiBenedetto — APPROVED** [review]

File: runtime/src/lib.rs (hunk 1 above)
Comment (drgora) [reply runtime/src/lib.rs:398]: Not quite, `MAX_AUTHORITIES` accounts for all validators, that is active + eligible, whereas `MaxActiveValidators` accounts only for the subset of active validators.
They are different things, so I'd rather keep them separated in the code, even if in the current configuration we use the same value for both.

File: runtime/src/lib.rs [hunk 2]
```diff
@@ -131,7 +131,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // The version of the runtime specification. A full node will not attempt to use its native
     //   runtime in substitute for the on-chain Wasm runtime unless all of `spec_name`,
     //   `spec_version`, and `authoring_version` are the same between Wasm and native.
-    spec_version: 2_000,
+    spec_version: 2_100,
```
Comment (la10736) [inline runtime/src/lib.rs:134]: We should change the runtime `Cargo.toml` too

**Review by la10736 — CHANGES_REQUESTED** [review]
Just change also cargo runtime version too

File: runtime/src/lib.rs (hunk 2 above)
Comment (drgora) [reply runtime/src/lib.rs]: Done, thanks!

File: runtime/src/lib.rs (hunk 1 above)
Comment (drgora) [reply runtime/src/lib.rs]: I have renamed `MAX_AUTHORITIES` and added some comments, I hope it's more intuitive now

**Review by la10736 — APPROVED** [review]
LGTM

**Review by ptagl — APPROVED** [review]

**Pull request merged**